Sources/protoc-gen-swift/ExtensionSetGenerator.swift

```diff
@@ -226,6 +236,12 @@ class ExtensionSetGenerator {
     func generateProtobufExtensionDeclarations(printer p: inout CodePrinter) {
       guard !extensions.isEmpty else { return }
 
+      p.print(
+          "\n",
+          "// Extension Objects - The only reason these might be needed is when manually\n",
+          "// constructing a `SimpleExtensionMap`, otherwise, the above _Extension Properties_\n",
```

Comment:
I think there's a word missing here. "otherwise, _use_ the above..."?

Reply:
done